.github/workflows/nightly-build.yaml

```diff
@@ -63,20 +81,23 @@ jobs:
         with:
           file: config/dockerfiles/controller-manager/Dockerfile
           tags: ${{ steps.meta.outputs.tags }}
-          push: ${{ github.event_name != 'pull_request' }}
+          push: true
```

Comment:
It looks like we cannot push the image if this is a PR due to we cannot using the secret.